src/server/qgsserverrequest.h

```diff
@@ -129,7 +129,7 @@ class SERVER_EXPORT QgsServerRequest
 
     /**
     * Remove an header
-    * @param name
+    * \param name
```

Comment:
At some point I'll sit down and write a test if a param docstring consists only of "\param name" or "\param name the name" and propose to simply delete it in this case.